docs/get_started/version.md

```diff
@@ -0,0 +1,56 @@
+---
+id: version
+title: Version details
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Which version should be used?
+
+The recommendation is to always run the latest possible version of YuniKorn.
+New releases are made regular driven by new features for YuniKorn itself and or the shim. 
+Features and fixes are currently only added to the development branch from which releases are cut.
```

Comment:
Maybe we can highlight that "there is no patch version" currently. I guess most engineers are aware of semantic versioning :)

Reply:
added a line explaining that, in both english and zh-cn